src/Browscap/Data/Validator/ChildrenData.php

```diff
@@ -0,0 +1,236 @@
+<?php
+/**
+ * This file is part of the browscap package.
+ *
+ * Copyright (c) 1998-2017, Browser Capabilities Project
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types = 1);
+namespace Browscap\Data\Validator;
+
+use Browscap\Data\Helper\CheckDeviceData;
+use Browscap\Data\Helper\CheckEngineData;
+use Browscap\Data\Helper\CheckPlatformData;
+
+/**
+ * Class Children
+ *
+ * @category   Browscap
+ *
+ * @author     James Titcumb <james@asgrim.com>
+ */
+class ChildrenData
```

Comment:
Not quite sure what `ChildrenData` actually represents here? Can you comment in the classdoc to clarify please